client/src/app/create/create.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="panel panel-default">
+  <div class="panel-heading"><h4>Translate given word</h4></div>
+  <div class="panel-body ">
+    <div class="row center-block">
+      <form class="col-xs-12 col-sm-8 col-sm-offset-2" (ngSubmit)="addNewWord()" [formGroup]="newWord">
```

Comment:
Nie wiem czy to dobry pomysł, ale proponuję żeby wszystkie rzeczy, które odnoszą się do angulara w templacie, pisać w single quote. To będzie taki znacznik i od razu będziesz wiedział że dana właściwość, odnosi się do zmiennej komponentu. Powiedz co o tym myślisz